.github/workflows/release-ebpf-profiler.yaml

```
@@ -0,0 +1,16 @@
+name: Release eBPF Profiler
+
+on:
+  push:
+    tags: ["v*"]
+
+jobs:
+  release:
+    name: Release eBPF Profiler
+    uses: ./.github/workflows/base-release.yaml
+    with:
+      distribution: otelcol-ebpf-profiler
+      goos: '[ "linux" ]'
+      goarch: '[ "amd64", "arm64", "ppc64le", "s390x" ]'
```

The profiler receiver doesn't work on `ppc64le` and `s390x` architectures.

```suggestion
      goarch: '[ "amd64", "arm64" ]'
```